proxy/src/redis/notifications.rs

```diff
@@ -243,29 +243,30 @@ impl<C: ProjectInfoCache + Send + Sync + 'static> MessageHandler<C> {
 
 fn invalidate_cache<C: ProjectInfoCache>(cache: Arc<C>, msg: Notification) {
     match msg {
-        Notification::AllowedIpsUpdate { allowed_ips_update } => {
-            cache.invalidate_allowed_ips_for_project(allowed_ips_update.project_id);
+        Notification::AllowedIpsUpdate {
+            allowed_ips_update: AllowedIpsUpdate { project_id },
```

Comment:
It is not clear to me why we don't invalidate the project cache here. Is it intentional?

Reply:
We are invalidating the cache. This is an "or pattern" with the following line. So if it's allowips or block that is updated it invalidates the same.

Reply:
Oh, I see! Sorry, reviewing in github UI sometimes is a bit challenging